stix_shifter_modules/aws_athena/stix_translation/README.md

```diff
@@ -0,0 +1,422 @@
+```
+
+#### Transmit query: VPCFlow query is passed to STIX transmission module
+
+```
+transmit aws_athena "{\"region\": \"us-east-1\", \"s3_bucket_location\": \"s3://queryresults-athena-s3/\", 
+\"vpcflow_database_name\":\"logs_db\", \"vpcflow_table_name\":\"vpc_flow_log\", 
+\"guardduty_database_name\":\"logs_db\",\"guardduty_table_name\":\"gd_logs\"}" "{\"auth\":{\"aws_access_key_id\": \"xxxx\", 
+\"aws_secret_access_key\": \"yyyy\"}}" query "{\"vpcflow\": \"((lower(sourceaddress) = lower('172.31.76.105') OR lower
+(destinationaddress) = lower('172.31.76.105')) AND starttime BETWEEN 1588322590 AND 1606732990) LIMIT 10000\"}"
+```
+
+#### GuardDuty Search id: 
+
+```
+ {'success': True, 'search_id': 'd512d194-396e-4afb-9e38-4011c2472edc:guardduty'}
+```
+
+#### VPCFlow Search id:
+
+```
+{'success': True, 'search_id': 'c3be3246-8b2b-4be7-b2de-d5d475c0ed8a:vpcflow'}
+```
+
+#### GuardDuty Transmit result: (provide search id in transmit result with offset and length)
+
+```
+ transmit aws_athena "{\"region\": \"us-east-1\", \"s3_bucket_location\": \"s3://queryresults-athena-s3/\", 
+ \"vpcflow_database_name\":\"logs_db\", \"vpcflow_table_name\":\"vpc_flow_log\", 
+ \"guardduty_database_name\":\"logs_db\",\"guardduty_table_name\":\"gd_logs\"}" "{\"auth\":{\"aws_access_key_id\": \"xxxx\", \"aws_secret_access_key\": \"yyyy\"}}" 
+ results "d512d194-396e-4afb-9e38-4011c2472edc:guardduty" 0 2
+```
+
+#### VPCFlow Transmit result : (provide search id in transmit result with offset and length)
+
+```
+transmit aws_athena "{\"region\": \"us-east-1\", \"s3_bucket_location\": \"s3://queryresults-athena-s3/\", 
+\"vpcflow_database_name\":\"logs_db\", \"vpcflow_table_name\":\"vpc_flow_log\", 
+\"guardduty_database_name\":\"logs_db\",\"guardduty_table_name\":\"gd_logs\"}" "{\"auth\":{\"aws_access_key_id\": \"xxxx\", \"aws_secret_access_key\": \"yyyy\"}}" 
+results "19b3ad96-a8de-4894-bb9d-f63c50c99b0a:vpcflow" 0 2
+```
+
+#### GuardDuty STIX observable output:
+
+```json
+ {
+    "type": "bundle",
+    "id": "bundle--0b9321b6-deba-4b89-a4dc-44f50ec3cf8c",
+    "spec_version": "2.0",
+    "objects": [
+        {
+            "type": "identity",
+            "id": "identity--f431f809-377b-45e0-aa1c-6a4751cae5ff",
+            "name": "aws_athena",
+            "identity_class": "events"
+        },
+        {
+            "id": "observed-data--3e13aa4b-2047-4082-8ff5-94ff9178f2f7",
+            "type": "observed-data",
+            "created_by_ref": "identity--f431f809-377b-45e0-aa1c-6a4751cae5ff",
+            "created": "2020-11-16T16:04:11.955Z",
+            "modified": "2020-11-16T16:04:11.955Z",
+            "objects": {
+                "0": {
+                    "type": "x-aws-details",
+                    "account_id": 979326520502,
+                    "region": "us-east-1"
+                },
+                "1": {
+                    "type": "x-ibm-finding",
+                    "finding_id": "24bae74503647e535aa16eeb246de329",
```

Comment:
You'll want to remove the finding id from the example bundle if removing it from the mapping

Reply:
@delliott90 The finding id is now removed from the README samples and the code is updated in PR branch